media-sound/fluidsynth/fluidsynth-2.0.4.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-multilib flag-o-matic
+
+DESCRIPTION="Software real-time synthesizer based on the Soundfont 2 specifications"
+HOMEPAGE="http://www.fluidsynth.org/"
+SRC_URI="https://github.com/FluidSynth/fluidsynth/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1+"
+SLOT="0/2"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd"
+IUSE="alsa dbus debug examples ipv6 jack ladspa lash oss portaudio pulseaudio +readline +sndfile"
+
+RDEPEND="
+	dev-libs/glib[${MULTILIB_USEDEP}]
+	alsa? (
+		media-libs/alsa-lib[${MULTILIB_USEDEP}]
+		lash? ( media-sound/lash[${MULTILIB_USEDEP}] )
+	)
+	dbus? ( sys-apps/dbus[${MULTILIB_USEDEP}] )
+	jack? ( virtual/jack[${MULTILIB_USEDEP}] )
+	ladspa? (
+		media-libs/ladspa-cmt[${MULTILIB_USEDEP}]
+		media-libs/ladspa-sdk[${MULTILIB_USEDEP}]
+	)
+	portaudio? ( media-libs/portaudio[${MULTILIB_USEDEP}] )
+	pulseaudio? ( media-sound/pulseaudio[${MULTILIB_USEDEP}] )
+	readline? ( sys-libs/readline:0=[${MULTILIB_USEDEP}] )
+	sndfile? ( media-libs/libsndfile[${MULTILIB_USEDEP}] )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig[${MULTILIB_USEDEP}]
```

Comment:
this needs to go into `BDEPEND` instead for EAPI-7.

Reply:
Done. I also swapped RDEPEND and DEPEND, which seems more natural and is used in examples in the EAPI 7 documentation.

I also removed the forced `-pthread` flag. Bug 436762 is from 2012 and it seems it has been fixed long ago already in the build system.